tests/containers/nfs/run_nfs.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+set -uxo  pipefail
+
+function start()
+{
+
+    # prepare /etc/exports
+    for i in "$@"; do
+        # fsid=0: needed for NFSv4
+        echo "$i *(rw,fsid=0,insecure,no_root_squash)" >> /etc/exports
+        echo "Serving $i"
+    done
+
+    # start rpcbind if it is not started yet
+    /usr/sbin/rpcinfo 127.0.0.1 > /dev/null; s=$?
+    if [ $s -ne 0 ]; then
+       echo "Starting rpcbind"
+       /usr/sbin/rpcbind -w
+    fi
+
+    mount -t nfsd nfsd /proc/fs/nfsd
+
+    # -V 3: enable NFSv3
+    /usr/sbin/rpc.mountd -N 2 -V 3
+
+    /usr/sbin/exportfs -r
+    # -G 10 to reduce grace time to 10 seconds (the lowest allowed)
+    /usr/sbin/rpc.nfsd -G 10 -V 3
+    /usr/sbin/rpc.statd --no-notify
+    echo "NFS started"
+}
+
+function stop()
+{
+    echo "Stopping NFS"
+
+    /usr/sbin/rpc.nfsd 0
+    /usr/sbin/exportfs -au
+    /usr/sbin/exportfs -f
+
+    kill "$( pidof rpc.mountd )"
+    umount /proc/fs/nfsd
+    echo > /etc/exports
+    exit 0
+}
+
+# rpc.statd has issues with very high ulimits
+ulimit -n 65535
+
+trap stop TERM
+
+start "$@"
+
+set +x
+# Ugly hack to do nothing and wait for SIGTERM
+while true; do
+    sleep 5
+done
```

Comment:
we use a while loop here to get log messages to show a heartbeat essentially (that it's not frozen)? Might also be nice to get a timestamp printed too.